src/components/ListItem/__tests__/__snapshots__/listItem.jest.js.snap

```diff
@@ -1,71 +1,76 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly with disabled 1`] = `
-<div
+<li
   aria-disabled={true}
   className="typography primary start singleLineEllipsis small normal listItem small disabled"
   data-testid="list-item"
+  id="undefined-item-undefined"
```

Comment:
mistake?

Reply:
No, cause the formula for `ListItem` id is `${listId}-item-${index}` and in this snapshot we are rendering just ListItem, without a `List`, so it doesn't receive `listId` or `index`